_posts/blog/2018-09-21/2018-09-21-3-things-that-make-it-really-easy-to-shop-in-openbazaar.markdown

```diff
@@ -0,0 +1,53 @@
+---
+title: "3 Things That Make It <span class='post-title-extra'>Really Easy to Shop</span> in OpenBazaar"
+layout: post
+date: '2018-09-21 06:30:00 -0300'
+social_title: '3 Things That Make It Really Easy to Shop in OpenBazaar'
+social_description: 'OpenBazaar is a decentralized online marketplace. OB1, the team leading the development, has designed a few things that make it easier to use.'
+social_image: Search Providers in OpenBazaar.gif.png
+---
+
+OpenBazaar is not like a regular online marketplace.
+
+It’s a _decentralized_ online marketplace.
```

Comment:
Maybe we should coin the term "market network"?

Reply:
I could dig it...